src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -655,34 +657,34 @@ export class TerminalService implements ITerminalService {
 
 	public focusFindWidget(): Promise<void> {
 		return this.showPanel(false).then(() => {
-			const panel = this._panelService.getActivePanel() as TerminalPanel;
-			panel.focusFindWidget();
+			const pane = this._viewsService.getActiveViewWithId(TERMINAL_VIEW_ID) as TerminalViewPane;
+			pane.focusFindWidget();
 			this._findWidgetVisible.set(true);
 		});
 	}
 
 	public hideFindWidget(): void {
-		const panel = this._panelService.getActivePanel() as TerminalPanel;
-		if (panel && panel.getId() === TERMINAL_PANEL_ID) {
-			panel.hideFindWidget();
+		const pane = this._viewsService.getActiveViewWithId(TERMINAL_VIEW_ID) as TerminalViewPane;
+		if (pane) {
+			pane.hideFindWidget();
 			this._findWidgetVisible.reset();
-			panel.focus();
+			pane.focus();
 		}
```

Comment:
This and findNext/findPrevious no longer verify if the terminal is active or not, I guess that's fine though.

Reply:
getActiveViewWithId only returns the requested view if it is active. so if (pane) is sufficient